internal/databases/mongo/binary/storage_local.go

```diff
@@ -120,8 +118,8 @@ func (localStorage *LocalStorage) CreateDirectories(directoryMetas []*BackupDire
 		if err != nil {
 			return err
 		}
-		userName := "mongodb"  // todo: get from sentinel!!!
-		groupName := "mongodb" // todo: get from sentinel!!!
+		userName := "mongodb"
```

Comment:
Why not get it from sentinel? Maybe introduce a config option? Also, maybe just remove the chown logic completely and run WAL-G from the appropriate user?

Reply:
it is legacy, need only to restore from old binary backup without tar archive. I want delete all legacy code.